query-connector/src/app/constants.ts

```diff
@@ -8,69 +8,35 @@ import {
   MedicationAdministration,
   MedicationRequest,
 } from "fhir/r4";
-/**
- * The use cases that can be used in the app
- */
-export const UseCases = [
-  "social-determinants",
-  "newborn-screening",
-  "syphilis",
-  "gonorrhea",
-  "chlamydia",
-  "cancer",
-] as const;
-export type USE_CASES = (typeof UseCases)[number];
-
-export const UseCaseToQueryName: {
-  [key in USE_CASES]: string;
-} = {
-  "social-determinants": "Gather social determinants of health",
-  "newborn-screening": "Newborn screening follow-up",
-  syphilis: "Syphilis case investigation",
-  gonorrhea: "Gonorrhea case investigation",
-  chlamydia: "Chlamydia case investigation",
-  cancer: "Cancer case investigation",
-};
-
-/**
- * Labels and values for the query options dropdown on the query page
- */
-export const demoQueryOptions = [
-  { value: "cancer", label: "Cancer case investigation" },
-  { value: "chlamydia", label: "Chlamydia case investigation" },
-  { value: "gonorrhea", label: "Gonorrhea case investigation" },
-  { value: "newborn-screening", label: "Newborn screening follow-up" },
-  // Temporarily remove social determinants
-  // {
-  //   value: "social-determinants",
-  //   label: "Gather social determinants of health",
-  // },
-  { value: "syphilis", label: "Syphilis case investigation" },
-];
 
-type DemoQueryOptionValue = (typeof demoQueryLabels)[number];
-export const demoQueryValToLabelMap = demoQueryOptions.reduce(
-  (acc, curVal) => {
-    acc[curVal.value as DemoQueryOptionValue] = curVal.label;
-    return acc;
+export const USE_CASE_DETAILS = {
+  "social-determinants": {
```

Comment:
I don't think we want to actually include social determinants anymore; this was a more custom query that we haven't fully supported as of yet (so if they try to customize it, they get 0 things to customize). Or we could comment it out for now to hedge.